installer/bundle/bundle_skel_Linux.sh

```diff
@@ -330,20 +330,18 @@ shouldInstall_scx()
 
 remove_and_install()
 {
-    check_if_pkg_is_installed apache-cimprov
-    if [ $? -eq 0 ]; then
-        pkg_rm apache-cimprov
-    fi
-    check_if_pkg_is_installed mysql-cimprov
-    if [ $? -eq 0 ]; then
-        pkg_rm mysql-cimprov
-    fi
-    check_if_pkg_is_installed scx
-    if [ $? -eq 0 ]; then
+    if [ -f /opt/microsoft/scx/bin/uninstall ]; then
```

Comment:
2012R2 does not seem to have uninstall script. Since we did release UR13 for 2012 R2 few days back, I think we should keep the fall back code for now.